src/originalBudgetTrackingApp/eventEmitters/EventEmitter.ts

```diff
@@ -19,14 +19,15 @@ export enum EventNames {
   LOG = 'LOG'
 }
 
-interface ErrorEvent {
-  error: Error
-}
-
 interface BudgetTrackingEvent {
   message?: string;
 }
 
+// TODO: Maybe extend Error?
```

Comment:
Why do we need to extend error? It has an error inside. This is an event so we want it to behave like other events. So I think it should extend BudgetTrackingEvent

Reply:
I thought about that I think because then you getting the `Error.message`, but nevermind, we decided to continue to work to improve these types.